packages/libs/react-ui/src/components/Form/Form.css.ts

```diff
@@ -1,20 +1,73 @@
+import { colorPalette } from '@theme/colors';
 import { sprinkles } from '@theme/sprinkles.css';
 import { darkThemeClass, vars } from '@theme/vars.css';
-import { style } from '@vanilla-extract/css';
+import { createVar, fallbackVar, style } from '@vanilla-extract/css';
+
+export type Status = 'disabled' | 'positive' | 'warning' | 'negative';
+export const statusColor = createVar();
+export const statusOutlineColor = createVar();
 
 export const baseOutlinedClass = style([
   sprinkles({
     borderRadius: '$sm',
   }),
   {
-    outline: `2px solid ${vars.colors.$gray30}`,
-    outlineOffset: '1px',
+    outline: `2px solid ${fallbackVar(statusColor, vars.colors.$gray30)}`,
+    outlineOffset: '2px',
+    selectors: {
+      [`${darkThemeClass} &`]: {
+        outline: `2px solid ${fallbackVar(statusColor, vars.colors.$gray60)}`,
+      },
+    },
+  },
+]);
+
+export const statusClass = style([
+  sprinkles({
+    borderRadius: '$sm',
+  }),
+  {
+    outline: `2px solid ${fallbackVar(
+      statusOutlineColor,
+      vars.colors.$blue60,
+    )}`,
+
+    outlineOffset: '2px',
```

Comment:
I think you can move this outlineOffset style to the baseContainerClass and then it should be applied regardless of the state